custom/opm/beneficiary.py

```diff
@@ -564,6 +564,24 @@ def weight_grade_normal(self):
                     return self.child_age/12
                 return False
 
+    def weight_grade_status(self, status):
+        if self.status == 'pregnant':
+            return False
```

Comment:
I don't think this matters as this method is only used in those list comprehensions below,  but if the status is pregnant, the condition wasn't not met, it just doesn't apply.  Elsewhere in these reports the convention is:
  `None` - not applicable
  `True` - applicable and condition met
  `False` - applicable but condition not met
And since `None` is falsy, it would work the same in your comprehensions.